app/src/main/java/com/example/learn09082021v001/MainActivity.kt

```diff
@@ -9,64 +9,69 @@ import androidx.activity.result.ActivityResult
 import androidx.activity.result.contract.ActivityResultContracts
 import kotlinx.android.synthetic.main.activity_data_input.*
 import kotlinx.android.synthetic.main.activity_main.*
-public var nameAndroid = "nameAndroidView"
-public var scoreAndroid = "scoreAndroidView"
-public var nameIos = "nameIosView"
-public var scoreIos = "scoreIosView"
+import java.io.Serializable;
+
+public val sendData = "datatranfer"
+public var scoreOOP = NameScoreOOP()
```

Comment:
this is variable, not a constant, should declare it inside MainActivity

Reply:
because inside Mainactivity only use Mainactivity : i need use 3 activity